packages/lab/src/Flow/nodes/DashboardNode.styles.tsx

```diff
@@ -0,0 +1,15 @@
+import { createClasses, theme } from "@hitachivantara/uikit-react-core";
+
+import { staticClasses as nodeClasses } from "../Node/Node.styles";
+
+const baseClasses = Object.fromEntries(
+  Object.keys(nodeClasses).map((key) => [key, {}])
+) as Record<keyof typeof nodeClasses, {}>;
```

Comment:
Just an heads-up: this means we'll be getting duplicated classes. eg.
```ts
// before
"HvFlowBaseNode-contentContainer"
// now
"HvFlowBaseNode-contentContainer HvFlowNode-contentContainer HvDashboardNode-contentContainer"
```

But let's address this another time

Reply:
Yes, I noticed this. We can look at this after, I couldn't find another way to enable users to override the parents classes.